tools/block-generator/generator/utils.go

```diff
@@ -79,3 +82,12 @@ func encode(handle codec.Handle, obj interface{}) ([]byte, error) {
 	}
 	return output, nil
 }
+
+func readGenesis(filepath string) (genesis bookkeeping.Genesis, err error) {
+	genesisText, err := os.ReadFile(filepath)
```

Comment:
```suggestion
	genesis, err = os.ReadFile(filepath)
```
Did you mean this? Otherwise, `genesis` is never set.

Reply:
NVM. I see it now. Resolving.